.github/workflows/.releaserc

```diff
@@ -0,0 +1,16 @@
+---
+branches:
+- master
+plugins:
+- - "@semantic-release/commit-analyzer"
+  - preset: conventionalcommits
+- - "@semantic-release/release-notes-generator"
+  - preset: conventionalcommits
+- - "@semantic-release/github"
+  - assets:
```

Comment:
Do we really want to have the converter as a binary?

Reply:
No opinion really. I added it as it is already there so just wanted to make sure we still publish it.